tools/gulp/tasks/unit-test.ts

```diff
@@ -48,20 +48,20 @@ gulp.task(':test:deps:inline', sequenceTask(':test:deps', ':inline-resources'));
  *
  * This task should be used when running unit tests locally.
  */
-gulp.task('test', [':test:watch'], (done: () => void) => {
+gulp.task('test', [':test:watch'], () => {
   new karma.Server({
     configFile: path.join(PROJECT_ROOT, 'test/karma.conf.js')
-  }, done).start();
+  }).start();
```

Comment:
This is probably not the right fix. By default, Karma uses `process.exit` as the callback. We definitely want to exit Gulp gracefully.